CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.5.1] - 2018-09-12
+### Changed
+- Change example text on modal by the official text.
```

Comment:
This should be in Spanish

Reply:
I dont know why you can not see it. If you go to Files Changes you could check changes are there.